OpenRVS/classes/OpenMultiPlayerWidget.uc

```diff
@@ -165,18 +167,35 @@ function GetGSServers()
 		NewItem.Created();
 		NewItem.iMainSvrListIdx = i;
 		NewItem.bFavorite = true;
-		NewItem.bSameVersion = true;
+		NewItem.bSameVersion = !ServerList[i].bWrongVersion;//true;//1.5 change
 		NewItem.szIPAddr = ServerList[i].IP;
-		NewItem.iPing = 1000;//was 1000 in early versions, at some point post 0.6 was changed to 9000?
+		if ( ( ServerList[i].iPing != 0 ) && ( ServerList[i].iPing != -1 ) && ( ServerList[i].iPing != 1000 ) )//1.5 change
+			NewItem.iPing = ServerList[i].iPing;
+		else
+		{
+			ServerList[i].iPing = 1000;//for sorting - null value is 0 which sorted as better than low ping servers

```

Comment:
When sorting: ints default to 0, but that would sort as BETTER ping than eg. a server with 30 ping.  So we modify the ServerList entry to 1000 so that sorting would treat this ping as bad.